reformCage.py

```diff
@@ -0,0 +1,177 @@
+"""
```

Comment:
The file should be called reform_cage.py not reformCage.py to keep a consistent style with the repo and comply with python style guidelines.